app/controllers/course/forum/posts_controller.rb

```diff
@@ -28,6 +29,7 @@ def update
 
   def vote
     @post.cast_vote!(current_user, post_vote_param)
+    Course::Forum::PostNotifier.post_voted(current_user, @post)
```

Comment:
OK, think we need a discussion regarding the up/down vote activity. 

I moved this to forum controller because there are quite a few kinds of posts, not only forum posts.